<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_remove_tour_page</class>
 <widget class="QDialog" name="add_remove_tour_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>602</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontal_4">
    <property name="spacing">
     <number>50</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLineEdit" name="add_team_name_input">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>: Team Name :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="remove_team_name_input"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>681</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="all_tour_list"/>
    </item>
    <item>
     <widget class="QListWidget" name="own_tour_list"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="all_tour_lable">
      <property name="text">
       <string>All  Tournoments :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="own_tour_lable">
      <property name="text">
       <string>Tournoments you are in :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>602</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontal_3">
    <property name="spacing">
     <number>50</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLineEdit" name="add_tour_name_input">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>: Tournoment name :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="remove_tour_name_input"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
